deploy/examples/cluster.yaml

```diff
@@ -19,9 +19,9 @@ spec:
     # v15 is octopus, and v16 is pacific.
     # RECOMMENDATION: In production, use a specific version tag instead of the general v14 flag, which pulls the latest release and could result in different
     # versions running within the cluster. See tags available at https://hub.docker.com/r/ceph/ceph/tags/.
-    # If you want to be more precise, you can always use a timestamp tag such quay.io/ceph/ceph:v16.2.7-20220216
+    # If you want to be more precise, you can always use a timestamp tag such quay.io/ceph/ceph:v17.2.0-20220216
```

Comment:
```suggestion
    # If you want to be more precise, you can always use a timestamp tag such quay.io/ceph/ceph:v17.2.0-20220420
```

Reply:
Done, where to find this timestamp? https://ceph.io/en/news/blog/2022/v17-2-0-quincy-released/

Reply:
quay.io/ceph/ceph